HoneyDo.Web/ClientApp/src/components/core/DatetimePicker.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component, SyntheticEvent } from "react";
+import Flatpickr from "react-flatpickr";
+import "flatpickr/dist/themes/light.css";
```

Comment:
Is this being built in webpack really?

Reply:
Yep... I don’t like it. But whatever.

Reply:
Oh really, why not? I'm just surprised, I've had issues with webpack and css files with react.